app/helpers/users_helper.rb

```diff
@@ -15,4 +15,11 @@ def user_lock_status(user)
       'Active'
     end
   end
+
+  def role_toggle_button(user, new_role)
```

Comment:
This is pretty cool, does this dynamically creates the button text based on roles?

Reply:
yep! Spits out a button that, when you click it, patches to those endpoints.